### PR TITLE
Add HDM to installer

### PR DESCRIPTION
**Pull request by tuxmea**

### CODE REVIEW

File: config/foreman.migrations/20240219155518_hdm.rb
Comment (ekohl) [inline config/foreman.migrations/20240219155518_hdm.rb:1]: Should be
```ruby
answers['foreman::plugin::hdm'] ||= false
answers['foreman_proxy::plugin::hdm'] ||= false
```

File: config/foreman-proxy-content.migrations/240219155741-hdm.rb
Comment (ekohl) [inline config/foreman-proxy-content.migrations/240219155741-hdm.rb:1]: Should be
```ruby
answers['foreman_proxy::plugin::hdm'] ||= false
```

**Review by ekohl — CHANGES_REQUESTED** [review]
I'm missing a katello migration.

**Comment by ekohl** [comment]
CI will fail until https://github.com/theforeman/puppet-foreman_proxy/pull/829 is merged.

**Review by ekohl — COMMENTED** [review]
It would be nice if the timestamps of the migrations matched, to hint that they're for the same content. We have [bin/create-migration](https://github.com/theforeman/foreman-installer/blob/develop/bin/create-migration) to create the migrations that does this.

Other than that, this looks correct.

File: config/foreman.migrations/20240219155518_hdm.rb
Comment (tuxmea) [reply config/foreman.migrations/20240219155518_hdm.rb:1]: Done

File: config/foreman-proxy-content.migrations/240219155741-hdm.rb
Comment (tuxmea) [reply config/foreman-proxy-content.migrations/240219155741-hdm.rb:1]: Done

**Comment by ekohl** [comment]
Ok to test

**Comment by tuxmea** [comment]
Can someone please approve the workflow? Thank you.

**Comment by ekohl** [comment]
@tuxmea done, but the tests aren't happy.

File: spec/fixtures/pulpcore-migration/katello-answers-after.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ foreman::plugin::dhcp_browser: false
 foreman::plugin::expire_hosts: false
 foreman::plugin::git_templates: false
 foreman::plugin::google: false
+foreman::plugin::hdm: false
```
Comment (evgeni) [inline spec/fixtures/pulpcore-migration/katello-answers-after.yaml:25]: these changes need to happen also to the other existing `pulpcore-*` fixtures, then the tests will pass

Comment (ekohl) [reply spec/fixtures/pulpcore-migration/katello-answers-after.yaml:25]: I've been debating if we need to drop the test - at this point it's not giving us anything and only causing overhead.

**Comment by tuxmea** [comment]
@ekohl Commits are squashed and tests are still green ;-)

**Comment by tuxmea** [comment]
@ekohl Is this OK to merge or do we have to wait for some other works to finish?

**Review by ekohl — APPROVED** [review]
Nope, all good now. I just missed the notification.

**Pull request merged**